### PR TITLE
[14.0][IMP] base_tier_validation, add 'Todo by' column in review table

### PR DESCRIPTION
**Pull request by kittiu**
This PR add new column that allow users to see who is next in approve line.

image

### CODE REVIEW

**Review by etobella — APPROVED** [review]
LGTM

File: base_tier_validation/models/tier_review.py [hunk 1]
```diff
@@ -77,5 +78,10 @@ def _compute_reviewer_ids(self):
         for rec in self:
             rec.reviewer_ids = rec._get_reviewers()
 
+    @api.depends("reviewer_ids")
+    def _compute_todo_by(self):
+        for rec in self:
+            rec.todo_by = ", ".join(rec.reviewer_ids.mapped("display_name"))
```
Comment (LoisRForgeFlow) [inline base_tier_validation/models/tier_review.py]: I'm not sure I like this, if you make tiers by groups, the list can be massive. Some ideas:
1. You could show the group name if it is a tier by group.
2. You can truncate the list if too long. E.g: "James Jamison, Daniel Davidson, Jacks Jackson (and 12 more .).

Comment (kittiu) [reply base_tier_validation/models/tier_review.py]: Agree with option 2, I will revise it a bit for that. Thanks for your review.

Comment (kittiu) [reply base_tier_validation/models/tier_review.py]: image

Comment (kittiu) [reply base_tier_validation/models/tier_review.py]: Done! if it is a group, show group name, if it is a list show abbreviation list.

Comment (etobella) [reply base_tier_validation/models/tier_review.py]: :+1:

Comment (LoisRForgeFlow) [reply base_tier_validation/models/tier_review.py]: That is great, thanks!

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM

**Pull request merged**